frontend/src/pages/settings/settings.html

```diff
@@ -1,4 +1,4 @@
-<div>
+<div v-if="!loading">
```

Comment:
This will cause the full page to fully reaload every time a button action is used. The loading boolean is mostly there to implement a loading animation if needed. I grant you that it is not implemented yet and useless for now. But I would not put it there, it is computing heavy for no gain (correct me if I'm wrong). Is it somehow needed?